www/creating-zooming-images.html

```diff
@@ -98,7 +98,19 @@ <h2>Creating Zooming Images</h2>
         <td><a href="/examples/tilesource-dzi/">DZI</a></td>
     </tr>
     <tr>
-        <td colspan="3"><h4>IIIF</h4></td>
+        <td colspan="3"><h4>IIIF</h4>
+            (requires a IIIF-compliant image server with JPEG 2000 support, like <a href="https://github.com/pulibrary/loris">Loris</a>; see also <a href="http://iiif.io">iiif.io</a> for more details)
```

Comment:
Actually this comment only applies to Kakadu; the other two are servers themselves.